dockstore-webservice/src/test/java/io/swagger/api/impl/ToolsImplCommonTest.java

```diff
@@ -26,6 +27,11 @@
  * @since 17/01/18
  */
 public class ToolsImplCommonTest {
+    static final Gson gson = new Gson();
+    static final String REFERENCE1 = "aaa";
+    static final String REFERENCE2 = "bbb";
+    static final String REFERENCE3 = "ccc";
+    static String json;
```

Comment:
Are these used in more than one test? 
Is there a reason for these to be static?